### ISSUE TITLE
First-time users run into `chsh: option requires an argument -- 's'`

### ISSUE THREAD

**Issue by dandv** [robbyrussell/oh-my-zsh#3360]
I've just tried to install oh-my-zsh for the first time on a new machine:

```sh
$ curl -L http://install.ohmyz.sh | sh
[...]
Cloning Oh My Zsh...
Cloning into '/home/dan/.oh-my-zsh'...
[...]
Looking for an existing zsh config...
Using the Oh My Zsh template file and adding it to ~/.zshrc
Copying your current PATH and adding it to the end of ~/.zshrc for you.
Time to change your default shell to zsh!
chsh: option requires an argument -- 's'
Usage: chsh [options] [LOGIN]

Options:
  -h, --help                    display this help message and exit
  -s, --shell SHELL             new login shell for the user account

$ chsh -s zsh
Password:
chsh: zsh is an invalid shell
$ chsh -s /bin/zsh
Password:
chsh: /bin/zsh is an invalid shell
````

Perhaps the automated installer should complain that `zsh` wasn't install, and offer to install it?

**Comment by brunobatista** [comment]
+1

**Comment by tobiasmuecksch** [comment]
+1

**Comment by mharmuth** [comment]
+1

**Comment by drawcard** [comment]
+1

**Comment by linux-modder** [comment]
+1 to non issue  zsh is an expected depsolv of oh-my-zsh

**Comment by chelogui** [comment]
+1

**Comment by wjwwood** [comment]
It should at least print out the command it meant to run so that users can recover easily from it.

It could be as easy as:

```sh
zsh --version > /dev/null 2>&1
if [ $? -eq 0 ]; then
  echo "running: chsh -s zsh"
  chsh -s zsh
else
  echo "zsh does not appear to be installed."
  echo "please install it and then run: chsh -s zsh"
fi
```

Would a pull request like this be considered (obviously after being cleaned up)?

**Comment by drawcard** [comment]
@wjwwood Great idea :+1:

**Comment by wjwwood** [comment]
> @wjwwood Great idea :+1:

https://github.com/robbyrussell/oh-my-zsh/pull/3799

**Comment by mcornella** [comment]
Fixed in #3809.
